contracts/container/contract.go

```diff
@@ -613,6 +614,18 @@ func Get(containerID []byte) Container {
 	return cnt
 }
 
+// GetContainer returns binary of the container it was created with by ID.
+//
+// If the container is missing, GetContainer throws [cst.NotFoundError]
+// exception.
+func GetContainer(id []byte) []byte {
```

Comment:
Maybe `GetContainerData`? Eventually I expect #449 to be implemented, it can use a different name of course, but this one fits better for it.

Reply:
good idea. `GetEACLData` too?

Reply:
Yes.

Reply:
done